ct/cmd/lint.go

```diff
@@ -69,6 +69,8 @@ func addLintFlags(flags *flag.FlagSet) {
 			Enable schema validation of 'Chart.yaml' using Yamale (default: true)`))
 	flags.Bool("validate-yaml", true, heredoc.Doc(`
 			Enable linting of 'Chart.yaml' and values files (default: true)`))
+	flags.StringSlice("additional-commands", []string{}, heredoc.Doc(`
+			Additional command to run per chart (default: [])`))
```

Comment:
We should add some more documentation for this.

Reply:
Good point. I added some.